### PR TITLE
zh-cn for Scala Tour: implicit-conversions.md

### PR DESCRIPTION
**Pull request by realwunan**
Simplified Chinese version of Scala Tour: Implicit Conversions

### CODE REVIEW

File: _zh-cn/tour/implicit-conversions.md [hunk 1]
```diff
@@ -13,3 +13,52 @@ language: zh-cn
...
+implicit def int2Integer(x: Int) =
+  java.lang.Integer.valueOf(x)
+```
+
+因为如果不加选择地使用隐式转换可能会导致陷阱，编译器会在编译隐式转换定义时发出警告。
+
+要关闭警告，执行以下任一操作：
+
+* 将 `scala.language.implicitConversions` 导入到隐式转换定义的上下文范围内
+* 将参数 `-language:implicitConversions` 传递给编译器
```
Comment (liufengyun) [inline _zh-cn/tour/implicit-conversions.md]: 启用编译器选项 `-language:implicitConversions`

File: _zh-cn/tour/implicit-conversions.md [hunk 2]
```diff
@@ -13,3 +13,52 @@ language: zh-cn
...
+
+implicit def int2Integer(x: Int) =
+  java.lang.Integer.valueOf(x)
+```
+
+因为如果不加选择地使用隐式转换可能会导致陷阱，编译器会在编译隐式转换定义时发出警告。
+
+要关闭警告，执行以下任一操作：
+
+* 将 `scala.language.implicitConversions` 导入到隐式转换定义的上下文范围内
```
Comment (liufengyun) [inline _zh-cn/tour/implicit-conversions.md]: `上下文范围` 翻得好 👍

Comment (realwunan) [reply _zh-cn/tour/implicit-conversions.md]: Thanks a lot.

File: _zh-cn/tour/implicit-conversions.md (hunk 1 above)
Comment (realwunan) [reply _zh-cn/tour/implicit-conversions.md]: It's better, thanks a lot.

**Comment by liufengyun** [comment]
@realwunan could you please try rebase against master to trigger CI again?

**Pull request merged**